ext/AvizExt/viz/scenarios.jl

```diff
@@ -157,9 +197,25 @@ function _render_scenarios_legend(
     # Add legend
     if get(opts, :legend, true)
         Legend(
-            g[1, 3], line_elements, labels; halign=:left, valign=:top, margin=(5, 5, 5, 5)
+            g[legend_position...],
+            line_elements,
+            labels;
+            halign=:left,
+            valign=:top,
+            margin=(5, 5, 5, 5),
         )
     end
 
     return nothing
 end
+
+# Sort types by variance in reverse order to plot highest variances first
+function _order_by_variance(
+    data::NamedDimsArray, scenario_types::NamedTuple
+)::Tuple{Symbol,Symbol,Symbol}
```

Comment:
Proper docstrings and blue style please

Reply:
About blue style:

1. This is in accordance to blue style guide 
![image](https://github.com/open-AIMS/ADRIA.jl/assets/8040719/b46ee701-4a79-450e-aeb7-1a371c9270dc)

2. I don't like it but, again, this is automatically done by the formatter... (are you starting to hate this formatter?)

Reply:
Okay with args being on same line as discussed. I think docstrings should be a proper docstring though, if that was part of your response.

Reply:
@ConnectedSystems I already changed the docstring